clients/da-vinci-client/src/main/java/com/linkedin/davinci/consumer/VeniceChangelogConsumerImpl.java

```diff
@@ -302,6 +303,36 @@ public CompletableFuture<Void> subscribeAll() {
     return this.subscribe(allPartitions);
   }
 
+  @Override
+  public CompletableFuture<Void> seekToTimestamp(Map<Integer, Long> timeStamps) {
```

Comment:
Nit:  `timestamps` instead of `timeStamps`. Or maybe the api name `seekToTimestamps` ?

Reply:
Sure